package.json

```diff
@@ -17,5 +17,9 @@
   },
   "dependencies": {
     "rxjs": "^5.0.2"
+  },
+  "devDependencies": {
+    "react-native": "^0.43.2",
```

Comment:
Yeah, that was I thought, we should not force using `react-native-windows`... Sure !

I'm agree for the devDependencies, but what about the peerDependencies ? Before I started to work on this project, the `react-native` plugin already was here, so why not the `react-native-windows` now ? Sorry, I don't understand all, I suppose we should specify what additional plugins is required for the `react-native-sensors` plugin right ?